test/test_torch.py

```diff
@@ -5404,6 +5404,229 @@ def test_narrow_empty(self, device):
             sz[d] = 0
             self.assertEqual(sz, y.size())
 
+    # This test calls torch.pad and numpy.pad many different ways to ensure
+    # that they give the same results
+    @onlyOnCPUAndCUDA
+    @dtypes(*torch.testing.get_all_dtypes(include_bfloat16=False))
```

Comment:
Why not bfloat16? The OpInfo includes it

Reply:
The only reason I disabled bfloat16 was that we're checking all the results against `numpy.pad`, which doesn't support bfloat16. I could add bfloat16 back in and check it through other means though

Reply:
That makes sense, I would just add a comment explaining that. This probably only needs to include a few dtypes. Maybe long, float32, complex64?

Reply:
Comment added. I can reduce the number of dtypes if you want